resources/assets/components/InboxItem/index.js

```diff
@@ -39,6 +41,22 @@ class InboxItem extends React.Component {
     const campaign = this.props.details.campaign;
     const signup = this.props.details.signup;
 
+    if (post['user']) {
```

Comment:
Hmm, this conditional bums me out a little. What are the two different scenarios that we get here?